apps/landingPage/src/components/demo.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Text, Heading } from '@kiwicom/orbit-components/lib/';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens'
+import { Link } from 'gatsby';
+
+export default ({iframeWidth, iframeHeight}) => (
+  <div style={styles.container}>
+    <div style={styles.container_dark}>
+      <Heading>
+        Just try it out!
+      </Heading>
+      <iframe
+        title="demo"
+        width={iframeWidth}
+        height={iframeHeight}
+        style={styles.iframe}
+        src="https://kiwicom-margarita.netlify.com/">
+      </iframe>
+      <div style={styles.container_text_bottom}>
+        <Heading type="title2">
+          Excited to know more!
+      </Heading>
+      <Text>
+        <Heading type="title2">
+          Just chekout the <a style={{color: 'black'}} href="https://github.com/kiwicom/margarita/">repository</a> or follow the <a href="https://github.com/kiwicom/margarita/#installing--getting-started"  style={{color: 'black'}}>instructions</a>.
+      </Heading>
+      </Text>
+      </div>
+    </div>
+    <Text size="large" italic={true}>
+      If you don't like <a href="https://tequila.kiwi.com/portal/login" style={{ color: defaultTokens.colorTextPrimary }}>Tequila</a> alone, grab a <Link to='/' style={{ color: defaultTokens.colorTextPrimary }}>Margarita</Link>
+    </Text>
+  </div>
+)
+
+const styles = {
```

Comment:
Also I see `styled-components` as one of the dependencies. Could we use that for styling instead of inline-styles?

Reply:
Ok! Changed all inline styles into styled components